catalog/transformer/catalog-transformer-generic-impl/src/main/java/ddf/catalog/transformer/generic/xml/impl/XMLSaxEventHandlerFactoryImpl.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.transformer.generic.xml.impl;
+
+import java.util.Map;
+
+import ddf.catalog.transformer.generic.xml.SaxEventHandler;
+import ddf.catalog.transformer.generic.xml.SaxEventHandlerFactory;
+
+public class XMLSaxEventHandlerFactoryImpl implements SaxEventHandlerFactory {
+
+    private static final String VERSION = "1.0";
+
+    private static final String ID = "XML_Handler";
```

Comment:
Service IDs usually are lower case and separate words with hyphens but in this case I think handler is incorrect since the service will be registered with the handler factory interface and xml is redundant since all SAX related event handlers and their factories are related to XML.

Looks like the handler mostly just supports the urn:catalog:metacard namespace.  Maybe the ID should be something like `catalog-metacard`.